test/unit/Reflection/ReflectionClassTest.php

```diff
@@ -680,6 +681,27 @@ public function testIsAnonymousWithAnonymousClassNoNamespace() : void
         self::assertStringEndsWith('Fixture/AnonymousClassNoNamespace.php(3)', $classInfo->getName());
     }
 
+    public function testIsAnonymousWithParentClass() : void
+    {
+        $reflector = new ClassReflector(
```

Comment:
Is this unused on purpose, or is the test incomplete?

Do you need any support to get this written?

Maybe just a reproducer and drop it to us to get the bug fixed?

Reply:
@Ocramius sorry, i was a bit busy last weeks, i will commit a test with your last suggestion about using `StringSourceLocator`, i will try to get this fixed by tomorrow, if am not able, i will close this PR and add the test code to the issue.